configs/recommended-module.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @fileoverview the `recommended-module` config for `eslint.config.js`
+ * @author 唯然<weiran.zsd@outlook.com>
+ */
+
+"use strict"
+
+const { configs, rules } = require("../lib/index.js")
+
+module.exports = {
+    plugins: { n: { rules } },
+    languageOptions: { sourceType: "module" },
```

Comment:
I intentionally only config `sourceType`, and no `ecmaVersion`, as I guess the `ecmaVersion: "latest"` sounds a good fit for most users.

The only problem: it's introduced in eslint v7.3x, while the plugin supports eslint>=7.0.0. Does it make sense to you? @mdjermanovic